% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/exception.R
\name{catch}
\alias{catch}
\title{Catch and execute function.}
\usage{
catch(value, catchfn)
}
\arguments{
\item{value}{Value which might be exception.}

\item{catchfn}{Function which will be called when expr throw exception with error message as argument.}
}
\value{
return value if expr don't throw, or exception if it throw.
}
\description{
Try execute function which might throw exception.
}
